frontend/src/Student/containers/PortfolioOtherMediaForm.js

```diff
@@ -0,0 +1,54 @@
+import { graphql } from 'react-apollo'
+import { push } from 'connected-react-router'
+import { connect } from 'react-redux'
+import { compose } from 'recompose'
+
+import { uploadImage, clearPreview } from '../actions'
+import { displayError} from '../../shared/actions'
+
+import CreatePortfolioOtherMedia from '../mutations/createPortfolioOtherMedia.graphql'
+import PeriodForSubmission from '../queries/periodForSubmission.graphql'
+import PortfolioByPeriod from '../queries/portfolioByPeriod.graphql'
+import PortfolioOtherMediaForm from '../components/PortfolioOtherMediaForm'
+
+const mapStateToProps = state => ({
+  previewFile: state.student.ui.submission.previewFile || {},
+  user: state.shared.auth.user
+})
+
+const mapDispatchToProps = dispatch => ({
+  done: () => dispatch(push('/portfolio')),
+  handlePDFUpload: file => dispatch(uploadPDF(file)),
+  handleImageUpload: file => dispatch(uploadImage(file)),
+  handleHometown: hometown => dispatch(setUserHometown(hometown)),
```

Comment:
Looking at photo submission it looks like it was removed there, we probably just forgot to take it out of this one. I can get it if you want since I think this was my file.

Reply:
We don't need to handle hometown on portfolios. Will take that out.